facebook-js-sdk/facebook-js-sdk.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for the Facebook Javascript SDK
+// Project: https://developers.facebook.com/docs/javascript
+// Definitions by: Amrit Kahlon <https://github.com/amritk/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import fb = facebook;
+declare var FB: fb.IFacebookStatic;
+declare module facebook {
+
+    interface IFacebookStatic {
```

Comment:
www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.